ui/vite.config.js

```diff
@@ -0,0 +1,65 @@
+import {defineConfig} from 'vite';
+import path from 'path';
+
+let build_config =
+  process.env.NODE_ENV === 'production'
+    ? {}
+    : {
+        lib: {
+          entry: '.',
+          name: 'gmf',
+        },
+      };
+
+export default defineConfig({
+  server: {
+    port: 3002,
+    //https: true,
+  },
+  base: '/static-frontend/',
+  build: {
+    lib: {
+      entry: '.',
+      formats: ['es'],
+    },
+    publicDir: true,
+    sourcemap: true,
+    rollupOptions: {
+      input: {
+        desktop: path.resolve(__dirname, 'desktop.html'),
+        mobile: path.resolve(__dirname, 'mobile.html'),
+        iframe_api: path.resolve(__dirname, 'iframe_api.html'),
+      },
+      output: {
+        inlineDynamicImports: false,
+      },
+    },
+    ...build_config,
+  },
+  resolve: {
+    alias: {
+      'ngeo': path.resolve(__dirname, 'node_modules/ngeo/distlib/src'),
+      'gmf': path.resolve(__dirname, 'node_modules/ngeo/distlib/src'),
+      'gmfapi': path.resolve(__dirname, 'node_modules/ngeo/distlib/srcapi'),
+      'api': path.resolve(__dirname, 'node_modules/ngeo/distlib/api'),
+      'jquery-ui/datepicker': path.resolve(__dirname, 'empty.js'), // For angular-ui-date
+      // Save about of 45k os bandwidth (gzipped) by unworking ignoring import done by a script downloaded from a CDN.
```

Comment:
This comment could be one line above, no ?

Reply:
The reason why I didn't put one line above is because if we solve the JQuery issue, the `jquery-ui/datepicker` still be necessary.
Is it OK like that, or you prefer that I move it one line above?

Reply:
No It's okay, I understand